sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -271,42 +275,45 @@ export class SessionReceiver {
 
   private _context: ClientEntityContext;
   private _receiveMode: ReceiveMode;
-  private _sessionId: string | undefined;
-  private _messageSession: MessageSession;
+  private _messageSession: MessageSession | undefined;
+  private _sessionOptions: SessionReceiverOptions;
 
   /**
    * @property {boolean} [isClosed] Denotes if close() was called on this receiver.
    * @readonly
    */
   public get isClosed(): boolean {
-    return !this._context.messageSessions[this.sessionId];
+    return this._messageSession ? !this._context.messageSessions[this.sessionId] : false;
   }
 
   /**
    * @property {string} [sessionId] The sessionId for the message session.
    * @readonly
    */
   public get sessionId(): string {
-    return this._sessionId || "";
+    return (this._messageSession && this._messageSession.sessionId) || "";
```

Comment:
Should the default be documented here? Is a session id of `""` a special value, or is it possible for a zero-length-named session to actually exist?

Reply:
empty string is not a valid sessionId.
Updated to return undefined instead.